javascript/javascript1/week2/index.js

```diff
@@ -0,0 +1,60 @@
+
+
+
+function getFullName(firstName, surName, formalName){
+    if(formalName===true){
+        return 'Lord!' +' ' + firstName +' ' +surName;
+    } else{
+         return firstName +' ' + surName;
+    }
+};
+console.log(getFullName('', '', ''));
+
+
+//Event application
+
+function weekdayEvent(event) {
+let weekDays= ['Sunday','Monday','Tursday','Wedensday','Thursday', 'Friday', 'Sutarday'];
+let today= new Date().getDay();
+const eventDay = (today+event) % weekDays.length;
+return weekDays[eventDay];
+
+}
+console.log(weekdayEvent(6));
+
+
+function wearSomeThing(temprature){
+    
+    if(temprature>=18){
+        console.log(" Wear T-Shert")
+    }else if(temprature< 18 && temprature>  15 ){
```

Comment:
No need to check `temperature < 18`. In this case is not needed.
```suggestion
    }else if( temprature>  15 ){
```